tinkerbell/stack/values.yaml

```diff
@@ -38,6 +38,7 @@ stack:
     # don't respect signals properly when run as PID 1.
     image: ghcr.io/jacobweinstock/dhcprelay # dhcprelay is a multiarch-enabled version of dhcrelay
     maxHopCount: 10
+    presentGIAddr: append
```

Comment:
This needs to be documented. Can we include an example of when and why its useful to change.